Docs/doc 1266 (#1286)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

### Step 3: Add a new SSH key to your GitHub account

> If you have not already configured your GitHub account to use an SSH key, you must do so in order to automatically download the GDK repositories using **InstallGDK.bat** as part of the next setup step.

Reviewer: ```suggestion
If you have not already configured your GitHub account to use an SSH key, you must do this in order to automatically download the GDK repositories as part of the next setup step.
```
Suggest "this" is less formal register.  Although "make it so" should be in the docs somewhere, IMHO.
Suggest we lose the batch file name as you don't need to know this yet. Cognitive overload during _Get started_.

> To do this:

Reviewer: <3

<br/>

------</br>

> _2019-08-03 Page updated with editorial review: added clarification on SSH key and Linux dependencies._

Reviewer: Added this direct to the branch - hope this is OK.

Reviewer: I know there have been other edits since May as I can see the sticking plasters in the text. :-)

> _2019-08-03 Page updated with editorial review: added clarification on SSH key and Linux dependencies._

Reviewer: ```suggestion
_2019-08-08 Page updated with editorial review: added clarification on SSH key and Linux dependencies._
```
